clang/test/SemaCXX/incomplete-array-extent.cpp

```diff
@@ -0,0 +1,5 @@
+// RUN: %clang_cc1 -verify -std=c++11 %s
+
+auto f() { // expected-error {{'auto' return without trailing return type; deduced return types are a C++14 extension}}
```

Comment:
I’d set the standard version to `C++14` or later to avoid unrelated diagnostics

Reply:
Thanks for the review. I've updated test